test/integration/bad/stanc.expected

```diff
@@ -2455,16 +2455,16 @@ Ill-typed arguments to '~' statement. No distribution 'ormal' was found with the
 
   $ ../../../../install/default/bin/stanc target-reserved.stan
 
-Syntax error in 'target-reserved.stan', line 2, column 2 to column 6, parsing error:
+Syntax error in 'target-reserved.stan', line 2, column 7 to column 13, parsing error:
    -------------------------------------------------
      1:  data {
      2:    real target;
-               ^
+                      ^
```

Comment:
How hard would it be to point this at the beginning of 'target' instead?

Reply:
Unfortunately I think pretty hard. The only way to get it to differentiate cases between keywords and anything else is to have the keyword on the stack when the error occurs, which means it points at one later token than the original case. We could change the error printing code to show the whole location span instead of the end of the span, but then I think other cases would underline the previous token (real in this case).